pruebas/package.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>pruebas</name>
+  <version>0.0.0</version>
```

Comment:
This seems to be the Semantic Versioning system. [It recommends](https://semver.org/lang/es/#c%C3%B3mo-deber%C3%ADa-lidiar-con-las-revisiones-en-la-fase-inicial-de-desarrollo-0yz) that we use `0.1.0` as the first version.

Reply:
done